packages/amplify-codegen/commands/codegen/add.js

```diff
@@ -1,14 +1,15 @@
-const codeGen = require('../../src/index')
+const codeGen = require('../../src/index');
 
-const featureName = 'add'
+const featureName = 'add';
 
 module.exports = {
   name: featureName,
   run: async (context) => {
     try {
-      await codeGen.add(context)
+      const apiId = context.parameters.options.apiId || null;
```

Comment:
`context.parameters.options` will always be defined. It might be an empty object but its always be present if the context is passed by gluegun

`apiId` can be undefined in which case it becomes null